.github/workflows/cockpit-lib-update.yml

```diff
@@ -0,0 +1,32 @@
+name: cockpit-lib-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
+  # can be run manually on https://github.com/cockpit-project/starter-kit/actions
+  workflow_dispatch:
+jobs:
+  cockpit-lib-update:
+    permissions:
+      contents: write
+      pull-requests: write
+      statuses: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y make
+
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
```

Comment:
We set up identity in other actions like this:
```
          git config user.name github-actions
          git config user.email github-actions@github.com
```
Alternatively, our bot identity is @InStallker, although we use it very little.

Would that work instead of cockpituous?

Reply:
Sure! This is just the default Cockpit uses.